src/serializer/ResidualHeapVisitor.js

```diff
@@ -113,20 +113,25 @@ export class ResidualHeapVisitor {
     }
   }
 
-  visitObjectProperties(obj: ObjectValue): void {
+  visitObjectProperties(obj: ObjectValue, kind?: ObjectKind): void {
     // visit properties
-    for (let [symbol, propertyBinding] of obj.symbols) {
-      invariant(propertyBinding);
-      let desc = propertyBinding.descriptor;
-      if (desc === undefined) continue; //deleted
-      this.visitDescriptor(desc);
-      this.visitValue(symbol);
+    if (kind !== "ReactElement") {
```

Comment:
Why ignore symbols? Ideally there probably shouldn't be any but if there are, they're probably ok visiting.

Reply:
If we visit it, then we need to handle it in serialisation stage by emitting it or adding it to the values (which I previously did, but removed due to prior feedback).

Reply:
ok. I still don't understand but maybe I missed some previous thread.